src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2370,11 +2445,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
-    Friend Partial Class BoundByRefArgumentWithCopyBack
+    Partial Friend Class BoundByRefArgumentWithCopyBack
         Protected Overrides Function ExpressionKind() As OperationKind
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
+            Get
+                Return ImmutableArray.Create(Of IOperation)(Me.OriginalArgument, Me.InConversion, Me.InPlaceholder, Me.OutConversion, Me.OutPlaceholder)
```

Comment:
OriginalArgument is fine.

---
In reply to: [113827208](https://github.com/dotnet/roslyn/pull/18976#discussion_r113827208) [](ancestors = 113827208)